ansible/roles/worker/templates/girder_worker.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Girder Worker Service
+After=network.target
+
+[Service]
+Type=simple
+User={{ worker_exec_user }}
+Group={{ worker_exec_group }}
+WorkingDirectory=/opt/girder_worker
+ExecStart=/bin/sh -c "LANG=en_US.UTF-8 python -m girder_worker --concurrency={{worker_concurrency}} --pidfile=.worker_pid --logfile=/opt/girder_worker/worker.log -l info"
```

Comment:
We have to have both Girder and Girder worker in a UTF8 locale.  We really don't care which one (I picked US English because of my own cultural bias).  The system locale defaults to `C`, which is *not* UTF8.  Without this, files that have non-ASCII characters (or maybe its non-LATIN1 characters), cannot be sent from Girder to girder_worker or back.